src/components/VmsPageHeader/index.js

```diff
@@ -23,7 +23,7 @@ const VmsPageHeader = ({ onRefresh }) => {
           <span className='icon-bar' />
           <span className='icon-bar' />
         </button>
-        <a href='#' className='navbar-brand obrand_headerLogoLink' id={`${idPrefix}-logo`}>
+        <a href='/' className='navbar-brand obrand_headerLogoLink' id={`${idPrefix}-logo`}>
```

Comment:
I believe `AppConfiguration.applicationURL` from `config.js` needs to be used since web-ui may run under different prefixes: `domain/` in dev mode, `domain/ovirt-engine/web-ui` in production mode.